src/game/default/g_item.h

```diff
@@ -45,6 +45,7 @@ const g_item_t *G_FindItem(const char *name);
 const g_item_t *G_FindItemByClassName(const char *class_name);
 const g_item_t *G_ItemByIndex(uint16_t index);
 const g_item_t *G_ClientArmor(const g_entity_t *ent);
+const g_armor_info_t *G_GetArmorInfo(const g_item_t *armor);
```

Comment:
Hate to be a style nazi, but we rarely use get/set nomenclature for functions. Mind naming this just `G_ArmorInfo`?